vagga.yaml

```diff
@@ -18,11 +19,6 @@ commands:
       env: !Command
         container: gdg_dev
         run: |
-          export OAUTHLIB_INSECURE_TRANSPORT=1
-          if [ ! -d "gdg.org.ua" ]; then
-          git clone https://github.com/GDG-Ukraine/gdg.org.ua
-          fi
           cd gdg.org.ua
-          make env
-          make dev-deps
+          sudo make dev-deps
```

Comment:
At this point I have following error:

npm http GET https://registry.npmjs.org/bower
npm http 304 https://registry.npmjs.org/bower
npm ERR! error rolling back Error: EROFS, unlink '/usr/local/lib/node_modules/bower'
npm ERR! error rolling back  bower@1.7.7 { [Error: EROFS, unlink '/usr/local/lib/node_modules/bower']
npm ERR! error rolling back   errno: 56,
npm ERR! error rolling back   code: 'EROFS',
npm ERR! error rolling back   path: '/usr/local/lib/node_modules/bower' }
npm ERR! Error: EROFS, unlink '/usr/local/lib/node_modules/bower'
npm ERR! If you need help, you may report this log at:
npm ERR!     <http://github.com/isaacs/npm/issues>
npm ERR! or email it to:
npm ERR!     <npm-@googlegroups.com>

npm ERR! System Linux 3.16.0-44-generic
npm ERR! command "/usr/bin/nodejs" "/usr/bin/npm" "install" "-g" "bower"
npm ERR! cwd /work
npm ERR! node -v v0.10.25
npm ERR! npm -v 1.3.10
npm ERR! path /usr/local/lib/node_modules/bower
npm ERR! code EROFS
npm ERR! errno 56
npm ERR! 
npm ERR! Additional logging details can be found in:
npm ERR!     /work/npm-debug.log
npm ERR! not ok code 0
make: *** [front-deps] Error 56

Probably, it is connected to not fully supported by npm virtualized filesystems. Maybe anyone have some suggestions? Actually, I'm interested in the following - do we actually need to install bower at 'run' stage? We can install it using NpmInstall, and here what is happening on _build container stage for this event:
Setting up nodejs-legacy (0.10.25~dfsg2-2ubuntu1) ...
npm http GET https://registry.npmjs.org/bower
npm http GET https://registry.npmjs.org/nodejs
npm http 304 https://registry.npmjs.org/bower
npm http 304 https://registry.npmjs.org/nodejs
/usr/local/bin/bower -> /usr/local/lib/node_modules/bower/bin/bower
nodejs@0.0.0 /usr/local/lib/node_modules/nodejs

bower@1.7.7 /usr/local/lib/node_modules/bower